extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/dashmanifestcreators/YoutubeDashManifestCreatorsUtils.java

```diff
@@ -298,8 +299,10 @@ public static void generateRoleElement(@Nonnull final Document doc,
             final Element roleElement = doc.createElement(ROLE);
 
             setAttribute(roleElement, doc, "schemeIdUri", "urn:mpeg:DASH:role:2011");
-            setAttribute(roleElement, doc, "value", itagItem.isDescriptiveAudio()
-                    ? "alternate" : "main");
+            setAttribute(roleElement, doc, "value",
+                    itagItem.getAudioTrackType() == null
+                            || itagItem.getAudioTrackType() == AudioTrackType.ORIGINAL
+                            ? "main" : "alternate");
```

Comment:
Could you apply the changes of #1039 here and use the correct property for dubbed tracks (see the standard in the PR I linked)?